packages/scheduler/lib/models/tasks.ts

```diff
@@ -226,7 +235,15 @@ export async function expiresIfTimeout(): Promise<Result<Task[]>> {
             .update({
                 state: 'EXPIRED',
                 last_state_transition_at: new Date(),
-                terminated: true
+                terminated: true,
+                output: db.raw(`
+                    CASE
```

Comment:
logic seems a bit verbose to be in a raw query. Unsure if there is a better way -- my only concern is maintenance

Reply:
I agree that it is verbose. Do you see a better way achieve the same thing?